packages/common/src/chains.ts

```diff
@@ -17,6 +17,22 @@ export enum ChainId {
   POLYGON_MUMBAI = 80001,
 }
 
+/**
+ * Attempts to parse a numerical or string chainId to the Enum.
+ * returns null if the chainid is invalid */
+export function tryParseChainIdToEnum(chainId: string | number) {
```

Comment:
Declaring the return type of the function would guard against accidentally returning `undefined` or something likewise unintended.